src/auto_scheduler/compute_dag.cc

```diff
@@ -324,17 +314,9 @@ String ComputeDAG::PrintStepsAsPython(const Array<Step>& transform_steps) const
          << "tuple(" << stage->op->name << ".op.reduce_axis)\n";
     }
   }
-  // Call each step's PrintAsPythonAPI method
+  // Call each step's ApplyToPythonAPI method
```

Comment:
`PrintAsPythonAPI` is a better name in my opinion

Reply:
Recover to PrintAsPythonAPI.